packages/@react-native-windows/tester/overrides.json

```diff
@@ -56,6 +56,13 @@
       "baseHash": "4ed1c701e08611ec3648e4dd27f5974c2a821a08",
       "issue": 6210
     },
+    {
+      "type": "derived",
```

Comment:
Nit: could you mark this as patch? At some point I'm wanting to do a cleanup so that we store all of the patch types as diffs.

Reply:
sure